layouts/joomla/edit/params.php

```diff
@@ -18,27 +18,27 @@
 $app       = Factory::getApplication();
 $form      = $displayData->getForm();
 $fieldSets = $form->getFieldsets();
-$helper    = ($displayData->useCoreUI ?? false) ? 'uitab' : 'bootstrap';
+$helper    = $displayData->get('useCoreUI', false) ? 'uitab' : 'bootstrap';
 
 if (empty($fieldSets)) {
     return;
 }
 
-$ignoreFieldsets      = $displayData->ignore_fieldsets ?? [];
-$outputFieldsets      = $displayData->output_fieldsets ?? [];
-$ignoreFieldsetFields = $displayData->ignore_fieldset_fields ?? [];
-$ignoreFields         = $displayData->ignore_fields ?? [];
-$extraFields          = $displayData->extra_fields ?? [];
-$tabName              = $displayData->tab_name ?? 'myTab';
+$ignoreFieldsets = $displayData->get('ignore_fieldsets') ?: [];
+$outputFieldsets = $displayData->get('output_fieldsets') ?: [];
+$ignoreFieldsetFields = $displayData->get('ignore_fieldset_fields') ?: [];
+$ignoreFields    = $displayData->get('ignore_fields') ?: [];
+$extraFields     = $displayData->get('extra_fields') ?: [];
+$tabName         = $displayData->get('tab_name') ?: 'myTab';
```

Comment:
```suggestion
$ignoreFieldsets      = $displayData->get('ignore_fieldsets') ?: [];
$outputFieldsets      = $displayData->get('output_fieldsets') ?: [];
$ignoreFieldsetFields = $displayData->get('ignore_fieldset_fields') ?: [];
$ignoreFields         = $displayData->get('ignore_fields') ?: [];
$extraFields          = $displayData->get('extra_fields') ?: [];
$tabName              = $displayData->get('tab_name') ?: 'myTab';
```
Assignment alignment.